index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
Hi @stephenmathieson I'm sorry we disagree here. Please support `null` or I'm not going to accept the PR. It works today with `null`. Please keep it working.